### ISSUE TITLE
Support python2.7.18

### ISSUE THREAD

**Issue by renyijiu** [crdoconnor/strictyaml#192]
* Argument

python2.7.18
strictyaml==1.6.2

* Bug

<img width="1150" alt="image" src="https://user-images.githubusercontent.com/8318266/221748698-25621910-d61d-4280-9ce2-3d6994818209.png">

When i test the package with python2.7.18, it raise error when import, so does this package support python2.7.18? if not, should update the setup.py.

**Comment by hyunlee1o** [comment]
Why would you still support python 2?

**Comment by crdoconnor** [comment]
yeah, I dropped python 2 support a while back but I think I might have forgotten to update the tags.
